html/is_valid_custom_element_name_test.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+import { assertEquals } from "@std/assert/equals";
+import { isValidCustomElementName } from "./is_valid_custom_element_name.ts";
+
+const forbiddenCustomElementNames: string[] = [
+  "annotation-xml",
+  "color-profile",
+  "font-face",
+  "font-face-src",
+  "font-face-uri",
+  "font-face-format",
+  "font-face-name",
+  "missing-glyph",
+] as const;
+
+Deno.test("isValidCustomElementName()", async (t) => {
+  await t.step("handles forbidden custom names", () => {
+    forbiddenCustomElementNames.map((forbiddenName) => {
+      assertEquals(isValidCustomElementName(forbiddenName), false);
+    });
+  });
+
+  await t.step("handles custom names with upper cases", () => {
+    assertEquals(isValidCustomElementName("Custom-element"), false);
+  });
+
+  await t.step("handles custom names with especial chars", () => {
+    assertEquals(isValidCustomElementName("custom-element@"), false);
+  });
+
+  await t.step("handles custom names with numbers", () => {
+    assertEquals(isValidCustomElementName("custom-1-element"), true);
+  });
+
+  await t.step("handles custom names with underscores", () => {
+    assertEquals(isValidCustomElementName("custom_element"), true);
```

Comment:
A custom element name needs to include at least one `-`. This shouldn't be a valid name.

Reply:
Hmmmm, you're right, just took a look at the HTML reference and got it. I'll fix it tomorrow afternoon